dev/test.sh

```diff
@@ -19,7 +19,7 @@ python -m black --exclude "src\/py\/flwr\/proto" --check src/py/flwr
 echo "- black: done"
 
 echo "- init_py_check: start"
-python -m flwr_tool.init_py_check src/py/flwr src/py/flwr_tool
+python -m src.py.flwr_tool.init_py_check src/py/flwr src/py/flwr_tool
```

Comment:
Can you elaborate on this line? It should work as is if `bootstrap.sh` runs correctly. It might be I am missing something.

Reply:
yes, the problem was that the poetry env was not sourced before running the tests. @achiverram28 , should we revisit the instructions about environment setup we provide for contributors? Is there something we should make more clear or even update?